### PR TITLE
Update Kaseya playbook

### PR DESCRIPTION
**Pull request by melamedbn**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Description
Update Kaseya playbook with:
* Kaseya VSA patch
* QRadar hunting - fix issues
* Link incidents - fix issues

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [ ] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-Kaseya_VSA__0-day_-_REvil_Ransomware_Supply_Chain_Attack.yml [hunk 1]
```diff
@@ -2481,7 +2496,335 @@ tasks:
...
+    skipunavailable: false
+    quietmode: 0
+  "84":
+    id: "84"
+    taskid: b4c43bd8-500a-4764-8900-ce1483a4ab34
+    type: playbook
+    task:
+      id: b4c43bd8-500a-4764-8900-ce1483a4ab34
+      version: -1
+      name: QRadarFullSearch
```
Comment (altmannyarden) [inline Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-Kaseya_VSA__0-day_-_REvil_Ransomware_Supply_Chain_Attack.yml:2595]: Need to change here as well

File: Packs/MajorBreachesInvestigationandResponse/ReleaseNotes/1_6_17.md [hunk 2]
```diff
@@ -0,0 +1,6 @@
+
+#### Playbooks
+##### Kaseya VSA  0-day - REvil Ransomware Supply Chain Attack
+- Added the patch released by Kaseya to mitigations
+- Added fixes to QRadar indicators hunting and Link Incidents
```
Comment (altmannyarden) [inline Packs/MajorBreachesInvestigationandResponse/ReleaseNotes/1_6_17.md:5]: Change Link Incidents to small letters

File: Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-Kaseya_VSA__0-day_-_REvil_Ransomware_Supply_Chain_Attack.yml (hunk 1 above)
Comment (melamedbn) [reply Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-Kaseya_VSA__0-day_-_REvil_Ransomware_Supply_Chain_Attack.yml:2595]: Can't find that in my yml

Comment (melamedbn) [reply Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-Kaseya_VSA__0-day_-_REvil_Ransomware_Supply_Chain_Attack.yml:2595]: fixed

File: Packs/MajorBreachesInvestigationandResponse/ReleaseNotes/1_6_17.md (hunk 2 above)
Comment (melamedbn) [reply Packs/MajorBreachesInvestigationandResponse/ReleaseNotes/1_6_17.md:5]: fixed

File: Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-Kaseya_VSA__0-day_-_REvil_Ransomware_Supply_Chain_Attack.yml [hunk 3]
```diff
@@ -921,10 +904,10 @@ tasks:
...
   "32":
     id: "32"
-    taskid: 3722321a-b4c7-4592-8183-dc500289e5f1
+    taskid: d181aa5d-1a23-4161-8201-85963b94379e
     type: playbook
     task:
-      id: 3722321a-b4c7-4592-8183-dc500289e5f1
+      id: d181aa5d-1a23-4161-8201-85963b94379e
       version: -1
       name: Panorama Query Logs
```
[the remark this answers is not in the document]
Comment (melamedbn) [reply Packs/MajorBreachesInvestigationandResponse/Playbooks/playbook-Kaseya_VSA__0-day_-_REvil_Ransomware_Supply_Chain_Attack.yml:912]: fixed

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**